composer.json

```
@@ -1,6 +1,6 @@
 {
     "name": "swag/swag-extension-store",
-    "version": "4.0.5",
+    "version": "4.0.6",
```

This should be a minor version:
```suggestion
    "version": "4.1.0",
```

---

Changed the version number to `4.1.0`